flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveTestUtils.java

```diff
@@ -108,4 +117,106 @@ public static TableEnvironment createTableEnv() {
 		tableEnv.getConfig().setSqlDialect(SqlDialect.HIVE);
 		return tableEnv;
 	}
+
+	// Insert into a single partition of a text table.
+	public static InsertTextTable insertToTextTable(HiveShell hiveShell, String dbName, String tableName) {
+		return new InsertTextTable(hiveShell, dbName, tableName);
+	}
+
+	/**
+	 * insert table operation.
+	 */
+	public static class InsertTextTable {
```

Comment:
We might want a better (using none rather than verb) class name, such as TextTableInserter or TextTableCreator.